scale/job/management/commands/scale_pre_steps.py

```diff
@@ -126,36 +122,14 @@ def _generate_input_metadata(self, job_exe):
                     input_metadata['RECIPE'][i] = [serialize(ScaleFile.objects.get_details(file_id=f)).data for f in
                                                    input_data.values[i].file_ids]
 
-        workspace_names = config.get_input_workspace_names()
-        workspace_models = {w.name: w for w in Workspace.objects.get_workspaces(names=workspace_names)}
+        logger.debug("Scale Input Metadata Manifest Generated:")
+        logger.debug(input_metadata)
```

Comment:
Minor formatting concern. This will output the dict, not json and you'll have all the unicode prefix mess in the output. Can you do something like here (https://github.com/ngageoint/scale/blob/e33d510d066ae8a033a863e983bdb1710b868336/scale/recipe/messages/process_condition.py#L94) to improve our sanity?